proposals/0052-svg-components.md

```diff
@@ -0,0 +1,247 @@
+<!--
+  Note: You are probably looking for `stage-1--discussion-template.md`!
+  This template is reserved for anyone championing an already-approved proposal.
+
+  Community members who would like to propose an idea or feature should begin
+  by creating a GitHub Discussion. See the repo README.md for more info.
+
+  To use this template: create a new, empty file in the repo under `proposals/${ID}.md`.
+  Replace `${ID}` with the official accepted proposal ID, found in the GitHub Issue
+  of the accepted proposal.
+-->
+
+**If you have feedback and the feature is released as experimental, please leave it on the Stage 3 PR. Otherwise, comment on the Stage 2 issue (links below).**
+
+- Start Date: 2024-10-01
+- Reference Issues: <!-- related issues, otherwise leave empty -->
+- Implementation PR: https://github.com/withastro/astro/pull/12067
+- Stage 2 Issue: https://github.com/withastro/roadmap/issues/699
+- Stage 3 PR: <!-- related roadmap PR, leave it empty if you don't have a PR yet -->
```

Comment:
We have a PR already, we can link it here.

Reply:
Ah, thanks!  I read this as the Stage 3 RFC PR link 😅